e2e/scenario_test.go

```diff
@@ -754,6 +754,10 @@ func Test_Ubuntu2204Gen2_ContainerdAirgappedK8sNotCached(t *testing.T) {
 						ContainerRegistryServer: fmt.Sprintf("%s.azurecr.io", config.PrivateACRName),
 					},
 				}
+				// intentionally using private acr url to get kube binaries
+				nbc.AgentPoolProfile.KubernetesConfig.CustomKubeBinaryURL = fmt.Sprintf(
+					"privateacre2ewestus3.azurecr.io/oss/binaries/kubernetes/kubernetes-node:v%s-linux-amd64",
```

Comment:
don't we set the private ACR base URL dynamically via config? if so we probably shouldn't hard-code it here

Reply:
Right the default one is set in `baseTemplateLinux` in node_config.go but I didn't want to change the base one, which is a acs-mirror url, because rp is still using acs-mirror. I think there is a plan to deprecate acs-mirror so by then we can update the `baseTemplateLinux`. What do you think?

Reply:
that makes sense, though I was talking about this specifically: https://github.com/Azure/AgentBaker/blob/4e6ec2f0758b3ad813a57de9051d77b9a46728b7/e2e/config/config.go#L21

can we just use that here instead? that way we'll theoretically be able to run this scenario in any region

Reply:
I got what you mean. I do this now.
```
nbc.AgentPoolProfile.KubernetesConfig.CustomKubeBinaryURL = fmt.Sprintf(
					"%s.azurecr.io/oss/binaries/kubernetes/kubernetes-node:v%s-linux-amd64",
					config.PrivateACRName,
					nbc.ContainerService.Properties.OrchestratorProfile.OrchestratorVersion)
```